languages/csharp/exercises/concept/chars/.docs/instructions.md

```diff
@@ -26,31 +26,27 @@ Identifier.Clean("my\0Id");
 
 ### 3. Convert kebab-case to camel-case
 
-An identifier such as my-object becomes myObject
+An identifier such as my-object becomes myObject.
 
 ```csharp
 Identifier.Clean("à-ḃç");
 // => "àḂç"
 ```
 
-### 4. Ensure that an already valid identifier remains unchnged by the routine
+### 4. Omit Greek lower case letters
 
-Note that the `Clean` method should treat an empty string as valid
+Omit any letters in the range 'α' to 'ω'.
 
 ```csharp
-Identifier.Clean("àḃç");
-// => "àḃç"
+Identifier.Clean("MyΟβιεγτFinder");
+// => "MyΟFinder"
 ```
 
-### 5. Insert character into a string
+### 5. Ensure that an already valid identifier remains unchnged by the routine
```

Comment:
Does this task require the student to implement new functionality? If not, it might be useful to have this just as a test.